src/modules/keyboardmanager/dll/Resources.resx

```diff
@@ -131,6 +131,7 @@
   </data>
   <data name="RegisterClassFailed_ErrorMessage" xml:space="preserve">
     <value>Windows registration failed!</value>
+    <comment>Windows as in the OS</comment>
```

Comment:
Not the OS here, it's supposed to refer to the display surface. I think it can be changed to `Window` actually.